modules/terraform/validate_struct.go

```diff
@@ -0,0 +1,62 @@
+package terraform
+
+import (
+	"errors"
+	"io/fs"
+	"os"
+	"path/filepath"
+
+	"github.com/gruntwork-io/terratest/modules/collections"
+)
+
+type ValidationOptions struct {
+	RootDir     string
+	ExcludeDirs []string
+}
```

Comment:
Add comments. Specifically, does `ExcludeDirs` need to be absolute, or relative to `RootDir`, or relative to current working dir to work?

Reply:
Clarified in 9519b48